Sources/Plasma/PubUtilLib/plPipeline/GL/plGLPipeline.cpp

```diff
@@ -1574,6 +1575,28 @@ void plGLPipeline::IPreprocessAvatarTextures()
 
 
 
+//// IIsViewLeftHanded ////////////////////////////////////////////////////////
+//  Returns true if the combination of the local2world and world2camera
+//  matrices is left-handed.
+
+bool plGLPipeline::IIsViewLeftHanded()
+{
+    return fView.GetViewTransform().GetOrthogonal() ^ ( fView.fLocalToWorldLeftHanded ^ fView.fWorldToCamLeftHanded ) ? true : false;
```

Comment:
This could probably live in plPipelineViewSettings as a public method, rather than needing to be implemented in both pipelines.

Reply:
Yeah. Code duplication feels like a thing right now. There are a lot of utility of scene functions that don't do any rendering. Folding them down into plPipeline3D or something other common class feels like a wider conversation.